t/27-consoles-sshXtermIPMI.t

```diff
@@ -56,14 +55,27 @@ $vnc_mock->noop('login');
 ok my $sol_connection = consoles::sshXtermIPMI->new($testapi_console, undef), 'sol connection can be established';
 
 subtest 'sshXtermIPMI activate' => sub {
+    my $ipc_run_mock = Test::MockModule->new('IPC::Run');
+    $ipc_run_mock->redefine(run => sub ($cmd, $stdin, $stdout, $stderr) {
+            $$stdin = 'in', $$stdout = 'out', $$stderr = 'err'; return 1;
+    });
     stderr_like {
         $sol_connection->activate();
     } qr/Xterm PID:/, 'VNC connection established';
+
+    $ipc_run_mock->redefine(run => sub ($cmd, $stdin, $stdout, $stderr) {
+            $$stdin = 'in', $$stdout = 'out', $$stderr = 'Unable to deactivate SOL payload'; return 0;
+    });
+    throws_ok {
+        $sol_connection->activate();
+    } qr/Unexpected IPMI response/, 'sshXterm dies with unexpected ipmi response';
+    $ipc_run_mock->unmock('run');
```

Comment:
Since the variable `$ipc_run_mock` falls out of scope after that subtest, it will automatically unmock

Reply:
idk why but it doesn't on my local environment.